Add cmd line switch to enable/disable acceleration emulation

broadwell opened the pull request:
If the `-e` switch is provided to `tiff2holes`, the note MIDI file will have tempo events added to emulate acceleration using the new algorithm from PR #13. Otherwise, the MIDI file will not accelerate and will remain at 60bpm for the entire duration. The raw MIDI file will never have acceleration-based tempo events applied to it.

broadwell committed:
Correct the data legend for the holes

broadwell committed:
Add cmd line switch to enable acceleration emulation

broadwell committed:
Set default tempo of 60 for unaccelerated note MIDI

simonwiles commented:
I've rebased this branch on `master` and once I've resolved the merge conflicts it's shaken down to three commits.  I'm pretty confident I've done it right, but rather than force push over this branch I've pushed the rebased version to `acceleration-switch-rebased`.  Could you just check it looks like you would expect it to?  Thanks!

broadwell committed:
Update the usage string to include the new flag.

This also adds the -s flag enabled in PR #15. Note that the Options library provides much more helpful usage inforation if the user specifies an unknown option (including "-h"), but it's not worth finessing this further.

broadwell commented:
Thanks for hammering out the `acceleration-switch-rebased` branch. I just took it for a spin, and it works as intended. I also added a new commit that updates the usage string of `tiff2holes`. Feel free to merge that branch (if you're comfortable with it), and retire this one.

I can handle the rebasing for the next PR unless you really want to -- also, there are a couple more in the pipeline behind it.

simonwiles commented:
No worries, thanks for giving it the okay.  I've just pushed the rebased branch over the top of this, and will now merge.
